src/core/globalstate.ts

```diff
@@ -88,7 +88,34 @@ export class MobXGlobals {
 
 export let globalState: MobXGlobals = new MobXGlobals();
 
+let shareGlobalStateCalled = false;
+let warnedAboutMultipleInstances = false;
+let _runInSandbox = false;
```

Comment:
For the sake of consistency, could `_runInSandbox` be named `runInSandboxCalled` just like `shareGlobalStateCalled`...?